### PR TITLE
Support for translating into another language  

### PR DESCRIPTION
**Pull request by unique1o1**

### CODE REVIEW

**Comment by gauthamzz** [comment]
Sorry for the delay, you changed too many lines. Idk even why you changed lines not related to your issues.

I need to test a bit and ill get back to you.

**Comment by lalitmee** [comment]
@gauthamzz, Actually there are these many changes because I think he used [Prettier](https://prettier.io/) for formatting the code. So, that is changing the formatting of the code which Prettier is having in `.prettierrc` or any other rules file.

And one thing I didn't understand, why he is changing the `Trigger word` **Hey** to **Ok**?

**Comment by gauthamzz** [comment]
yes thats true, thats why i didnt merge till now. There are too many changes.

File: background.js [hunk 1]
```diff
@@ -3,899 +3,1243 @@ chrome.browserAction.onClicked.addListener(function(tab) {
...
   var baseUrl = "https://api.api.ai/v1/";
   var talking = true;
   var recognition;
   var txt;
+
   var screenshotId = 1;
   var status = "active"; //for storing listening status
   var debug = false;
-  var our_trigger = "hey ";
+  var our_trigger = "ok ";
```
Comment (gauthamzz) [inline background.js:16]: please convert this back to hey

**Comment by gauthamzz** [comment]
This PR wont be merged due to following reasons 

1.  Inactivity
2. Large unwanted changes in the files 
3. When i tried the code, it broke my system and had to restart it.

I hope you understand, if you wish to make PR please only change your part. Many a commit for each change.

**Pull request closed without merging**